.gitignore

```diff
@@ -3,3 +3,4 @@ node_modules/
 *.zip
 dist/
 .release-notes/tmp*
+.vscode/*
```

Comment:
```suggestion
```

Unexpected change: did an automatic system added it?
=> self merging the suggestion